continuous_integration/envs/38-latest.yaml

```
@@ -5,12 +5,14 @@ dependencies:
   # required dependencies
   - python=3.8
   - dask=2021.05.0
+  - numba=0.53.1
   - distributed
   - geopandas
   - pygeos
   - pyproj=2.6  # unpin when pyproj 3.1 is released with multithreading fix
   # test dependencies
   - pytest
   - pytest-cov
+  - hilbert_curve=2.0.5
```

```suggestion
  - hilbertcurve
```

I think that this is the correct name. Also, no need to pin the version.

---

yes that's correct, my mistake